CONTRIBUTING.md

```diff
@@ -0,0 +1,248 @@
+# Contributing to Lux
+
+Thank you for your interest in contributing to Lux! This document will help
+answer any outstanding questions you may have about the contribution process.
+
+*Please read the [Code of Conduct](./CODE_OF_CONDUCT.md) before you participate
+in community.*
+
+## Contents
+
+*   [Reporting a Bug](#reporting-a-bug)
+*   [Requesting a Feature](#requesting-a-feature)
+*   [Development Workflow](#development-workflow)
+*   [Writing Documentation](#writing-documentation)
+*   [Submitting a Pull Request](#submitting-a-pull-request)
+*   [Helpful Links and Information](#helpful-links-and-information)
+
+## Reporting a Bug
+
+While bugs are unfortunate, they're a reality in software. We can't fix what we
+don't know about, so please report liberally. If you're not sure if something is
+a bug or not, feel free to file a bug anyway.
+
+If you have the chance, before reporting a bug, please search existing issues,
+as it's possible that someone else has already reported your error. This doesn't
+always work, and sometimes it's hard to know what to search for, so consider
+this extra credit. We won't mind if you accidentally file a duplicate report.
+
+Opening an issue is as easy as following [this link](https://github.com/postlight/lux/issues/new)
```

Comment:
This is a great suggestion. I included an adapted version of the language in the ThinkUp docs and included it in [CONTRIBUTING.md](https://github.com/postlight/lux/blob/docs-contributing/CONTRIBUTING.md#security). I also added *ThinkUp Security and Data Privacy* to the acknowledgements at the bottom of the document.